src/sass/main.scss

```diff
@@ -1,12 +1,47 @@
-/*
- * Główny plik dla stylów.
- * Wszystkie partials podłączamy tutaj.
- */
+/* 
+=====================================
+=====================================
+            IMPORT SASS
+=====================================
+=====================================
+*/
+/* IMPORT UTILTS 
+============================================= */
+@import './utils/variables';
 
-// Style dla pliku z przykładem HTML fragmentu
-@import './utils/media-query';
+/* IMPORT GENERAL
+============================================= */
 @import './general';
-@import './utils/variables';
-@import './example';
+
+/* IMPORT MEDIA
+============================================= */
+@import './utils/media-query';
+
+/* IMPORT SECTIONS
+============================================= */
+
+/* HEADER */
```

Comment:
u mnie wystarczy importować header-menu - pozostałe pliki scss są importowane do header menu - wtedy jedyne co aktualizuje to swój plik scss a main będzie miał ten sam stan

Reply:
Aha, więc u Ciebie "_header-menu.scss" importuje resztę plików z head?

Reply:
dokładnie - mniejszy ruch w main :)